[ADD] Attendances: new documentation needed for flow and configurations (#6497)
odoo/documentation: content/applications/hr/attendances.rst

   menu. The device then goes into kiosk mode.
#. Navigate to the :menuselection:`Attendances application --> Configuration`. In the
   :guilabel:`Kiosk Settings` section, click on the link in the :guilabel:`Attendance Kiosk Url`
   field. A new tab opens in kiosk mode. It is recommended to close the database tab to keep the
   database secure.
#. Navigate to the :menuselection:`Attendances application --> Configuration`. In the
   :guilabel:`Kiosk Settings` section, click :guilabel:`Copy` next to the link beneath the
   :guilabel:`Attendance Kiosk Url` field. Paste this url into a web browser on the kiosk device and
   navigate to it.

.. image:: attendances/kiosk-url.png
   :align: center
   :alt: The Attendances Kiosk URL field in the settings section of the Attendances application.

As a security measure, once a device is in kiosk mode, it is not possible to exit kiosk mode and go
back into the database without singing back in. To exit kiosk mode, click the back button in the web
browser. This logs the user out of the database, and returns to the main log in screen. This
prevents anyone from accessing the database, adding another layer of security.

Badge
-----

> To check in or out using a badge, tap the :guilabel:`📷 Tap to scan` image in the center of the

Reviewer: ```suggestion
To check in or out using a badge, tap the :guilabel:`📷 (camera) Tap to scan` image in the center of the
```

kiosk. Then, scan the barcode on the badge using the method configured in the :ref:`kiosk settings
<attendances/kiosk-settings>` section of the configuration menu. Options are a dedicated barcode
scanning device, or the kiosk's front or back camera. Once the barcode is scanned, the employee
is checked in or out, and a :ref:`confirmation message <attendances/confirmation>` appears with all
the check in or check out information.

.. image:: attendances/scan-badge.png
   :align: center
   :alt: The Attendances Kiosk view displaying the scan badge image.

RFID
----

To check in or out using an RFID key fob, simply scan the fob with an RFID reader. Once scanned, the
employee is then either checked in or checked out, and a :ref:`confirmation message
<attendances/confirmation>` appears with all the check in or check out information.

Manually
--------

> For user's who either do not have a scannable badge or an RFID fob, or simply do not have them when

Reviewer: ```suggestion
For users who either do not have a scannable badge or an RFID fob, or simply do not have them when
```

checking in our out, users can manually check in and out at a kiosk. Tap the :guilabel:`Identify
Manually` button on the kiosk, and a screen with all the employees that can be checked in or out
appears. This is the same view as in the *Employees* application dashboard. Tap on the person, and
they are either checked in or checked out, and a :ref:`confirmation message
<attendances/confirmation>` appears.

When a large number of employees appear in the list, making scrolling inefficient, there are two
ways to quickly find a specific person:

> - :guilabel:`Search...`: Tap on the :guilabel:`Search...` field and enter the person's name. As the

Reviewer: ```suggestion
- :guilabel:`Search...`: tap on the :guilabel:`Search...` field and enter the person's name. As the
```

  name is typed in, the matching results are displayed on the screen.

> - :guilabel:`Department`: To quickly filter the presented employees, tap on a department to display

Reviewer: ```suggestion
- :guilabel:`Department`: to quickly filter the presented employees, tap on a department to display
```

:guilabel:`(Employee) Welcome! Please enter your PIN to check in` appears above the numbers. When
checking out, :guilabel:`(Employee) Want to check out? Please enter your PIN to check out` appears
above the numbers. Tap in the PIN number using the number pad, then tap :guilabel:`OK` when done.
The employee is then checked in or out, and a :ref:`confirmation message
<attendances/confirmation>` appears.

.. image:: attendances/enter-pin.png
   :align: center
   :alt: The pop-up that appears when prompted to enter a pin number.

.. _attendances/confirmation:

Confirmation message
--------------------

When an employee checks in or out, a confirmation message appears with all the check in or check out
information. When checking in, a welcome message appears, as well as the date and time of check in.
An :guilabel:`Hours Previously Today: XX:XX` field also appears, displaying any time already logged
for that employee for the day. If no time has already been logged, the value displayed is `00:00`.
Beneath the message is an :guilabel:`OK` button. To exit the screen before the preset time in the
kiosk, tap the :guilabel:`OK` button.

> When checking out, the message displays a goodbye message, with the date and time of check out, and

Reviewer: ```suggestion
When checking out, the screen displays a goodbye message, with the date and time of check out, and
```

the total hours logged for the day. Beneath the message is a :guilabel:`Goodbye` button. To exit the
screen before the preset time, tap the :guilabel:`Goodbye` button.

.. image:: attendances/goodbye-message.png
   :align: center
   :alt: The goodbye message with all the employee's check out information.

Overview
========

When entering the *Attendances* application, the :guilabel:`Overview` dashboard is presented,
containing all the check in and check out information for the signed in user. If the user has
specific :ref:`access rights <attendances/access-rights>` and/or are :ref:`approvers
<attendances/approvers>` for specific employees, then those additional employee's check in and check

> out information is also visible on the :guilabel:`overview` dashboard.

Reviewer: ```suggestion
out information is also visible on the :guilabel:`Overview` dashboard.
```

Author: I wrote this with a capital O because on the menu at the top, the button to get to the Overview is with an "O", and I have it in a gui since I was sort of referencing the button, even though I didn't explicitly state that.

:guilabel:`Month`, or :guilabel:`Year` presented, click the left or right :guilabel:`arrow` buttons
on either side of the drop-down button. To jump back to a view containing the current day, click the
:guilabel:`Today` button, and the dashboard refreshes, presenting information containing the current
day's information.

In the :guilabel:`Day` view, the column for the current hour is highlighted in yellow. If the
:guilabel:`Week` or :guilabel:`Month` view is selected, the column for the current day is
highlighted. If the :guilabel:`Year` view is selected, the current month is highlighted.

.. image:: attendances/overview.png
   :align: center
   :alt: The overview dashboard presenting the information for the week, with the current day
         highlighted.

Any entries that have errors appear in red, indicating they need to be resolved by a user with the
proper :ref:`access rights <attendances/access-rights>` and/or are :ref:`approvers
<attendances/approvers>` for the employee(s) with the errors.

.. _attendances/filters-groups:

Filters and Groups
------------------

To filter the results in the dashboard or to present different groups of information, click the

> drop-down button in the :guilabel:`Search` bar above the dashboard, then select a filter or group.

Reviewer: ```suggestion
drop-down button in the :guilabel:`Search` bar above the dashboard, then select one of the available :guilabel:`Filters` or :guilabel:`Group By` options.
```

pop-up containing the details for that particular entry. To modify the :guilabel:`Check In` and/or
:guilabel:`Check Out` information, click on the :guilabel:`Check In` or :guilabel:`Check Out` field
and a calendar selector appears. Click on the desired date, then use the time selector beneath the
calendar to select the specific time for the entry. When the information is correct, click
:guilabel:`Apply.`

.. image:: attendances/errors.png
   :align: center
   :alt: The pop-up that allows for modifications to an attendance entry with an error. The calendar
         selector is shown, and the time selector is highlighted.

When all the information on the pop-up is correct, click :guilabel:`Save & Close`. If the entry no
longer has an error, the entry now appears in gray instead of red.

To delete an entry, click :guilabel:`Remove` on the pop-up instead of making modifications to the
entry.

Reporting
=========

To view attendance reports, click :guilabel:`Reporting` in the top menu. The default report displays
each employee's attendance information for the past 3 months, in a :guilabel:`Line Chart`.

> To default view is a :guilabel:`Graph`. To view the data in a pivot table, click the

Reviewer: ```suggestion
The default view is a :guilabel:`Graph`. To view the data in a pivot table, click the
```

Reviewer: I would move this below the section where you explain the different available graph views
